monkey/infection_monkey/credential_store/credentials_store.py

```diff
@@ -0,0 +1,29 @@
+from typing import Mapping
+
+from .i_credentials_store import ICredentialsStore
+
+
+class CredentialsStore(ICredentialsStore):
```

Comment:
In contrast to the interface, this concrete implementation *will* be couped to the ControlClient/Island in some way. We need a more descriptive name. What makes this credentials store different than other possible implementations? Maybe `AggregatingCredentialsStore`, since it aggregates crededentials from `add_credentials()` and from the control client.